src/mobile/apps/profile/routes.coffee

```diff
@@ -25,3 +25,7 @@ module.exports.setProfile = (req, res, next) ->
 
 module.exports.redirectEditorial = (req, res, next) ->
   res.redirect 301, req.url.replace 'editorial', 'articles'
+
+module.exports.redirectCollect = (req, res, next) ->
+  console.log("we are redirecting collect!")
+  next()
```

Comment:
Similarly wondering if the is necessary. Either way I think we will want to remove the `console.log`.